### PR TITLE
docs(tables): FRMW-266 Tables - Status Columns

### PR DESCRIPTION
**Pull request by ssalinas24**
JIRA: https://jira.rax.io/browse/FRMW-266

- [x] Dev LGTM
- [x] Dev LGTM
- [x] Design LGTM

### This JIRA Story Suggests the Following Copy:
"ENCORE-SPECIFIC: You can add consistency and color code your status columns using the CSS classes included in rxStatusColumn. Ticket Queues have their own styling that predates this and can be seen in their tickets.less file. However, if rxStatusColumn classes are not suitable, please send feedback through the "Submit feedback" link."

### Copy Recently Updated Partially in FRMW-236 Copy Edits as Follows: 
"ENCORE-SPECIFIC: You can add consistency and color code your status columns using the .rxStatusColumn class. Ticket Queues have their own styling that predates this and can be seen in their tickets.less file. Your new project should use rxStatusColumn, however, if
this is not suitable, send feedback."

### My Solution For This PR
I have made minor copy changes. Refer to the code in this PR.

### CODE REVIEW

**Comment by ssalinas24** [comment]
@CITguy This link produces a 404. Where is it supposed to go? `<a href="https://github.com/rackerlabs/encore-tq-ui/blob/master/app/styles/tickets.less">`

**Comment by glynnis** [comment]
LGTM

**Comment by CITguy** [comment]
@ssalinas24: I have absolutely no idea. It appears that ticketing has updated their file structure a bit (which broke the link). I'd be inclined to remove the reference to the ticketing repo.

**Comment by ssalinas24** [comment]
@CITguy I went ahead and removed the hyperlink and squashed. Ready for review to continue.

File: demo/styleguide/tables.html [hunk 1]
```diff
@@ -166,12 +166,11 @@ <h3 id="delete-x">Delete "X" action</h3>
             <span class="msg-info">ENCORE-SPECIFIC:</span>
             You can add consistency and color code your status columns using
             the <code>.rx-status-column</code> class. Ticket Queues have their
-            own styling that predates this and can be seen in their
-            <a href="https://github.com/rackerlabs/encore-tq-ui/blob/master/app/styles/tickets.less">
-              tickets.less
-            </a> file. Your new project should use
-            <a href="#/component/rxStatusColumn">rxStatusColumn</a>.
-            However, if this is not suitable, send feedback.
+            own styling that predates this and can be seen in their tickets.less
```
Comment (CITguy) [inline demo/styleguide/tables.html]: I'd remove the comment about `tickets.less`. That file doesn't seem to exist anymore.

```
Ticket Queues have their own styling that predates this.
```

Comment (ssalinas24) [inline demo/styleguide/tables.html]: @CITguy Ok I hear ya. Thanks for the insight. I will update.

**Comment by ssalinas24** [comment]
@CITguy  I have updated this to remove the mentioning of the less file.

**Comment by CITguy** [comment]
LGTM

**Comment by 100stacks** [comment]
LGTM

**Pull request merged**